samples/websocket-server/http.js

```diff
@@ -252,46 +337,34 @@ HttpServer.prototype = {
    */
   listen: function(port, opt_host) {
     var t = this;
-    socket.create('tcp', {}, function(socketInfo) {
-      t.socketInfo_ = socketInfo;
-      socket.listen(t.socketInfo_.socketId, opt_host || '0.0.0.0', port, 50,
-                    function(result) {
-        t.readyState_ = 1;
-        t.acceptConnection_(t.socketInfo_.socketId);
+    chrome.sockets.tcpServer.create(function(socketInfo) {
+      chrome.sockets.tcpServer.onAccept.addListener(function(acceptInfo) {
+        if (acceptInfo.socketId === socketInfo.socketId)
+          t.readRequestFromSocket_(new PSocket(acceptInfo.clientSocketId));
       });
+      
+      chrome.sockets.tcpServer.listen(
```

Comment:
When running this, I get this error below.
Please consider checking runtime.lastError. I get the "net::ERR_FILE_NOT_FOUND" error message as you can see.
```
extensions::lastError:133 Unchecked runtime.lastError while running sockets.tcpServer.listen: net::ERR_FILE_NOT_FOUND
    at Object.callback (chrome-extension://klacjkbkhfacpdolfbaehocfpjjipffa/http.js:346:32)